### PR TITLE
Issue 305 Adding a search on click to the tokens table

### PR DESCRIPTION
**Pull request by adarshram**
image

### CODE REVIEW

File: src/elements/tokens/TokenTable.tsx [hunk 1]
```diff
@@ -11,9 +11,16 @@ import { ReactComponent as IconSearch } from 'assets/icons/search.svg';
...
 
-export const TokenTable = () => {
+
+interface Props {
+  searchInput: string;
+  setSearchInput: Function;
+} 
+
+
+export const TokenTable = ({ searchInput,setSearchInput }: Props) => {
```
Comment (zavelevsky) [inline src/elements/tokens/TokenTable.tsx:21]: Please use Prettier to enforce code styling

Comment (adarshram) [reply src/elements/tokens/TokenTable.tsx]: done

**Review by zavelevsky — APPROVED** [review]
LGTM, we'll run it and test it

**Comment by zavelevsky** [comment]
fixes #305

File: src/elements/tokens/TopMovers.tsx [hunk 2]
```diff
@@ -4,9 +4,16 @@ import { Token } from 'services/observables/tokens';
...
-export const TopMovers = () => {
+interface Props {
+  setSearch: Function;
+}
+
+export const TopMovers = ({ setSearch }: Props) => {
   const tokens = useAppSelector<Token[]>((state) => state.bancor.tokens);
   const topTokens = orderBy(tokens, 'price_change_24', 'desc').slice(0, 20);
+  const handleClick = (token: Token) => {
+    setSearch(token.name);
```
Comment (zavelevsky) [inline src/elements/tokens/TopMovers.tsx:15]: expected: search input should contain the symbol

**Review by zavelevsky — CHANGES_REQUESTED** [review]
search by symbol, not name

File: src/elements/tokens/TopMovers.tsx (hunk 2 above)
Comment (adarshram) [reply src/elements/tokens/TopMovers.tsx]: done .

Comment (adarshram) [reply src/elements/tokens/TopMovers.tsx]: done .

**Review by zavelevsky — APPROVED** [review]
LGTM, thanks!

**Pull request merged**